src/app/tvshows/tvshows-search.component.ts

```diff
@@ -17,9 +17,7 @@ import { TvShowData } from './store/tvshows.models';
   <div *ngIf="selectors.isFetching$ | async">
     <progress value="80" max="100">fetching...</progress>
   </div>
-  <ul>
-    <li *ngFor="let show of (selectors.tvshows$ | async)">{{ show.title }}</li>
-  </ul>
+  <app-tvshow-poster *ngFor="let show of (selectors.tvshows$ | async)" [show]="show"></app-tvshow-poster>
```

Comment:
You might want to consider splitting the search and `results` into separate components and using a container to tie them together.

Reply:
Good idea. Will do.